src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ContentDialog/UnoSamples_Tests.ContentDialog.cs

```diff
@@ -350,5 +350,63 @@ public void ContentDialog_ComboBox()
 
 			_app.FastTap(closeButton);
 		}
+
+		[Test]
+		[AutoRetry]
+		public void ContentDialog_Simple_NotLightDismissible()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Controls.ContentDialogTests.ContentDialog_Simple");
+
+			var showDialogButton = _app.Marked("showDialog1");
+			var statusBarBackground = _app.Marked("statusBarBackground");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'statusBarBackground'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5106322)